### DISCUSSION TITLE
启动除gateway,monitor服务时报错

### DISCUSSION THREAD

**Discussion by pinkcao** [zuihou/zuihou-admin-cloud#178] [02.提问] [answered]
# 版本信息 （必填）：
后端使用的是lamp-cloud还是lamp-boot:  lamp-cloud-pro-datasource-column
前端使用的是lamp-web还是lamp-web-plus：lamp-web-pro
JDK 版本(必填) : 11
源码版本(必填)：4.11.0
MySQL(必填)：8.0.29
Nacos(必填)：2.0.2

# 问题描述：
启动BaseServerApplication时报错，提示Parameter 4 of constructor in top.tangyh.lamp.tenant.service.impl.DynamicDataSourceServiceImpl required a bean of type 'top.tangyh.lamp.tenant.mapper.InitDatabaseMapper' that could not be found 

# 报错截图
![buginfo](https://user-images.githubusercontent.com/36530450/187608435-23a1205c-7134-4b9c-9055-47f7174c4196.png)

# 重现步骤
原来使用4.10.0，于2022.8.30下午升级至4.11.0

1.拉取后端最新代码，install lamp-util-pro，install lamp-cloud-pro-datasource-column
2.拉取最新nacos配置，导入并修改修改数据库连接账号密码
3.启动BaseServerApplication，出现报错

**Comment by zuihou** [comment]
修改 database.yml  中的数据库连接信息

**Reply by pinkcao** [reply]
请问我应该修改哪一部分，我目前做了如下修改：
1.修改了mysql连接的url为具体的数据库地址
2.修改了mysql连接的username，password

**Comment by zuihou** [comment, marked as answer]
更新代码

**Reply by pinkcao** [reply]
成功跑起来了，但在BaseServerApplication、OauthServerApplication、SystemServerApplication启动过程中有报错：
![bugInfo2](https://user-images.githubusercontent.com/36530450/187631928-74d227c5-1e60-4d58-a75b-c566e94cf624.png)

**Reply by zuihou** [reply]
更新util-pro

**Reply by pinkcao** [reply]
ok，已正常运行
